packages/cloud2edge/requirements.yaml

```diff
@@ -12,8 +12,8 @@
 #
 dependencies:
   - name: hono
-    version: ^1.10.18
+    version: ~2.0.7
```

Comment:
@kalinkostashki the version range was before starting with `^` - I think we should keep it and not change to `~`

Reply:
ok, will fix this.